rkt/flag/secflags.go

```diff
@@ -19,19 +19,21 @@ const (
 	insecureImage = 1 << (iota - 1)
 	insecureTls
```

Comment:
Damn, this should really be "inscureTLS" ([c.f.](https://github.com/golang/go/wiki/CodeReviewComments#initialisms))